daemon-scheduler/pkg/regex/validate.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//     http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package regex
```

Comment:
Why name this package after its implementation technique?  Consumer of these functions cares that they validate or recognize particular string forms, not that they use regexps to do it (in fact they shouldn't know that).  Perhaps "validate", "sense", "recognize", or "awsnames" would be a more functional name for the module?